### PR TITLE
Us05 admin

### PR DESCRIPTION
**Pull request by Merhene**
- Added admin space :
   - Secured the new/edit/delete routes for GuestController, FlashController, TattooController
   - Added and personnalized EasyAdmin interface
   - Secured EasyAdmin route
- Updated grid to Flashes and tattoos
- Updated responsive for : 
    - Navbar
    - Media queries for cards
Finally erased dead codes in CSS and twig files

### CODE REVIEW

File: config/packages/security.yaml [hunk 1]
```diff
@@ -22,7 +34,7 @@ security:
     # Easy way to control access for large sections of your site
     # Note: Only the *first* access control that matches will be used
     access_control:
-        # - { path: ^/admin, roles: ROLE_ADMIN }
+         - { path: ^/admin, roles: ROLE_ADMIN }
         # - { path: ^/profile, roles: ROLE_USER }
```
Comment (benrcd) [inline config/packages/security.yaml]: Dead code

File: src/Controller/Admin/DashboardController.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    public function configureDashboard(): Dashboard
+    {
+        return Dashboard::new()
+            ->setTitle('2023 09 PHP LYON P3 Stan Dottori');
+    }
+
+    public function configureMenuItems(): iterable
+    {
+        // yield MenuItem::linkToDashboard('Dashboard', 'fa fa-home');
```
Comment (benrcd) [inline src/Controller/Admin/DashboardController.php]: Dead code

File: templates/base.html.twig [hunk 3]
```diff
@@ -8,13 +8,18 @@
 			{% block title %}Projet 3 Name
 			{% endblock %}
 		</title>
+        {# <nav>
+        {% include 'home/_navbar.html.twig' %}
```
Comment (benrcd) [inline templates/base.html.twig]: Dead code

File: templates/biography/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+					voluptate et animi atque et error temporibus quo quae velit nam nostrum internos
+					et possimus sunt. Ut quasi rerum eos voluptas vero qui quasi necessitatibus nam
+					dignissimos tempore qui dolorum cupiditate hic consequuntur esse non natus
+					pariatur. Et asperiores nihil id nemo tenetur vel obcaecati repudiandae rem
+					dignissimos numquam a explicabo tenetur sit galisum culpa. Sed labore pariatur et
+					voluptas soluta sit facilis blanditiis.
+			</p>
+		</div>
+		<br>
+		<div id="pagination">
```
Comment (benrcd) [inline templates/biography/index.html.twig:36]: Might as well remove the pagination now

File: templates/flash/index.html.twig [hunk 5]
```diff
@@ -7,22 +7,26 @@
...
+							<h6>Price:
+								{{ flash.price }}€</h6>
+							<h6>Available :
+								{{ flash.available ? 'Yes' : 'No' }}</h6>
+						</p>
+					</div>
+				</div>
+			</div>
+		{% else %}
+			<h3>no records found</h3>
```
Comment (benrcd) [inline templates/flash/index.html.twig]: Update the message to fit to the website

File: templates/guest/index.html.twig [hunk 6]
```diff
@@ -1,29 +1,29 @@
...
+					</p>
+				</div>
 			</div>
 		</div>
-                    <a href="{{ path('app_guest_show', {'id': guest.id}) }}">show</a>
-                    <a href="{{ path('app_guest_edit', {'id': guest.id}) }}">edit</a>
-        {% else %}
+		<a href="{{ path('app_guest_show', {'id': guest.id}) }}">show</a>
+	{% else %}
 		<h3>no records found</h3>
```
Comment (benrcd) [inline templates/guest/index.html.twig]: Update the message to fit to the website

File: templates/home/index.html.twig [hunk 7]
```diff
@@ -2,27 +2,52 @@
...
-
+{% block body_class %}specific-style
+{% endblock %}
 {% block body %}
-	<section class="home-scroll">
-		<div
-			class="stripe">
-			<img src="{{ asset('build/images/white_logo.png') }}" alt="">
+	<nav aria-label="Main navigation"><!-- Ceci n'est pas du code mort mais un outils ! -->
+		{# <div class="scroll-watcher"></div> #}
```
Comment (benrcd) [inline templates/home/index.html.twig:9]: We need to talk

File: templates/home/index.html.twig [hunk 8]
```diff
@@ -2,27 +2,52 @@
...
+				<div class="link" id="guest">
+					<a href="{{path('app_guest_index') }}">Guest</a>
+				</div>
+				<div class="link">Youtube</div>
+				<div class="link">Contact</div>
+			</section>
+		{% endblock %}
+	</main>
+	{% block javascript %}
+		<script>
```
Comment (benrcd) [inline templates/home/index.html.twig:47]: JS isn't supposed to be here

File: templates/security/login.html.twig [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+        <label for="password">Password</label>
+        <input type="password" name="_password" id="password" class="form-control" autocomplete="current-password" required>
+
+        <input type="hidden" name="_csrf_token"
+               value="{{ csrf_token('authenticate') }}"
+        >
+
+
+            {# Uncomment this section and add a remember_me option below your firewall to activate remember me functionality.
+            See https://symfony.com/doc/current/security/remember_me.html #}
```
Comment (benrcd) [inline templates/security/login.html.twig:29]: Is it done?

**Review by benrcd — CHANGES_REQUESTED** [review]

File: config/packages/security.yaml (hunk 1 above)
Comment (Merhene) [reply config/packages/security.yaml]: ok !

File: src/Controller/Admin/DashboardController.php (hunk 2 above)
Comment (Merhene) [reply src/Controller/Admin/DashboardController.php]: ok !

File: templates/base.html.twig (hunk 3 above)
Comment (Merhene) [reply templates/base.html.twig]: ok !

File: templates/biography/index.html.twig (hunk 4 above)
Comment (Merhene) [reply templates/biography/index.html.twig:36]: Will be done in an other US

File: templates/home/index.html.twig (hunk 8 above)
Comment (Merhene) [reply templates/home/index.html.twig:47]: Will be done in an other US

File: templates/security/login.html.twig (hunk 9 above)
Comment (Merhene) [reply templates/security/login.html.twig:29]: Yes it is ! 👍

**Review by benrcd — APPROVED** [review]

**Pull request merged**